.eslint-tailwindcss.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     ],
     settings: {
       tailwindcss: {
-        whitelist: ["(gc\\-).*"]
+        whitelist: ["(gc\\-).*", "form-builder", "page-container", "visually-hidden"]
```

Comment:
Does Tailwinds built in class of `sr-only` not meet the need for `visually-hidden`?

Reply:
This update isn't really in the scope of this PR but the update is more around getting rid of the lint issue vs needing the class.
<img width="869" alt="Screenshot 2023-07-12 at 11 11 38 AM" src="https://github.com/cds-snc/platform-forms-client/assets/62242/6b3a3351-8d19-4c3f-990d-97e190429f7f">

Reply:
Fair point.  I'll open a seperate issue that we can tackle later on.